package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "bedrock-web-payment",
+  "version": "0.0.1",
+  "description": "Client side utilities for bedrock-payment.",
+  "main": "lib/index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/digitalbazaar/bedrock-web-payment.git"
```

Comment:
For the publishing scripts to work properly, this needs to be an HTTPS url, removing the default `git+`

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-web-payment/pull/1/commits/0618c257af143d5a0adf0ea68f7f4f84c59f9358

and thanks for this one.